Quaver.Shared/Database/Playlists/PlaylistManager.cs

```diff
@@ -513,6 +528,62 @@ public static void UpdateMapInPlaylists(Map outdated, Map newMap)
             });
         }
 
+        /// <summary>
+        ///     Imports an online playlist
+        /// </summary>
+        /// <param name="onlineId"></param>
+        public static void ImportPlaylist(int onlineId)
+        {
+            Logger.Important($"Importing playlist: ID {onlineId}", LogType.Runtime);
+            var playlistResponse = new APIRequestPlaylistInformation(onlineId).ExecuteRequest();
+            if (playlistResponse == null || playlistResponse.Status != 200)
+            {
+                Logger.Important(
+                    $"Failed retrieving playlist information with error code: {playlistResponse?.Status ?? -1}",
+                    LogType.Network);
```

Comment:
Why is the formatting off on most of your logging calls? Should just be one line.